core/templates/dev/head/admin/Admin.js

```diff
@@ -23,6 +23,24 @@ oppia.controller('Admin', ['$scope', '$http', function($scope, $http) {
   $scope.adminTopicsCsvDownloadHandlerUrl = '/admintopicscsvdownloadhandler';
   $scope.configProperties = {};
 
+  $scope.currentTab = 'TAB_ACTIVITIES';
```

Comment:
Above this, declare named constants, and refer to them instead:

    $scope.TAB_ACTIVITIES = 'TAB_ACTIVITIES';

etc. It's a good practice that ensures you don't have skew in the strings. With the current approach, if a string is misspelled in the future, it probably won't be noticed; if we refer to constants instead, some kind of undefined error is likely to be raised.